src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -768,11 +761,18 @@ ExportQueryExecutionTrees::computeResultAsQLeverJSON(
     ++resultSize;
   }
 
+  RuntimeInformation runtimeInformation = qet.getRootOperation()->runtimeInfo();
+  runtimeInformation.addLimitOffsetRow(query._limitOffset, false);
+
   auto timeResultComputation =
       std::chrono::duration_cast<std::chrono::milliseconds>(
           timeUntilFunctionCall + runtimeInformation.totalTime_);
 
   nlohmann::json jsonSuffix;
+  jsonSuffix["runtimeInformation"]["meta"] = nlohmann::ordered_json(
+      qet.getRootOperation()->getRuntimeInfoWholeQuery());
+  jsonSuffix["runtimeInformation"]["query_execution_tree"] =
+      nlohmann::ordered_json(runtimeInformation);
```

Comment:
How complicated would it be to add a unit test that would have caught this regression?
In the meantime we have quite some experience in testing the exporting and the 
RuntimeInformation, so I think this could be doable.

Reply:
Done